wiki/People/The_Team/Beatmap_Nominators/Becoming_a_Beatmap_Nominator/es.md

```diff
@@ -1,5 +1,7 @@
 ---
 no_native_review: true
+outdated: true
+outdated_since: dc02d48ab01194487cf7bb5633c2982206fa8f53
```

Comment:
i dont think this should be added as actually there were no changes to the article

Reply:
true